src/tools/m65common.c

```diff
@@ -352,7 +353,7 @@ int stop_cpu(void)
   if (no_rxbuff)
     do_usleep(50000);
   cpu_stopped = 1;
-  slow_write_safe(fd, "t1\r", 3);
+  slow_write(fd, "t1\r", 3);
```

Comment:
Do check that this is reliable, as the slow_write_safe() routine purposely sends the characters slow enough that each ends up in a separate USB packet, and thus can't flood the MEGA65 bitstream if it doesn't have RX buffering on the uart monitor interface.

Reply:
This line came up on my radar due to a comment made by @frehwagen on Discord when he was discussing his F011 woes:
- https://discord.com/channels/719326990221574164/805492126128930846/832319254832873493

> OK, I tracked the issue down and it turned out to be an virtual f011 related issue at its core. While fixing it by synchronizing the M65 <-> HYPPO interaction I figured out that apperently stop_cpu() never actually stopped the CPU because using the safe_sent for "t1" turned the CPU on afterward again by sending "t0"

Given his finding, I reminded @frehwagen of it in a comment on his PR here, on line 338:
- https://github.com/MEGA65/mega65-tools/pull/54/files

I was surprised when he merged in his PR without changing the line to a logic that aligned with his discovery.

So I decided to rectify the matter on my card then.

For my part, I agree with what he discovered, that if the goal of the **stop_cpu()** was to enter trace mode by calling **slow_write_safe(fd, "t1\r", 3);**, then it would fail, since **slow_write_safe()**, does the following:
- do_slow_write() of t1 (enter trace mode)
- do_slow_write() of the data you requested
- **do_slow_write() of t0 (exit trace mode)**

So if your goal was to do **slow_write_safe("t1")**, in the hope of stopping the cpu, this won't go as planned, since your attempt to stop the cpu will be reverted by the last **do_slow_write()** call of **slow_write_safe()**.

As for your concern on the slow-sending of bytes, I think we're fine in that regard with the change above, as **slow_write()** does indeed do this already.

It's more that **slow_write_safe()** does this too, but also stops and starts the cpu along the way, as an extra duty.